db/migrate/20200907122828_create_issues.rb

```diff
@@ -0,0 +1,14 @@
+class CreateIssues < ActiveRecord::Migration[6.0]
+  def change
+    create_table :issues do |t|
+      t.string :title
+      t.text :description
+
+      t.integer :issue_type # enum: bug or issue
```

Comment:
add null: false to all the columns you have added validations for

Reply:
Doing it now

Reply:
Done